tests01.py

```diff
@@ -10,7 +10,7 @@ def test_string(self):
 
     def test_boolean(self):
         a = True
-        b = True
+        b = 3
```

Comment:
tests are failing here?
```suggestion
        b = True
```